models/bamf_nnunet_ct_kidney/config/slicer.yml

```diff
@@ -0,0 +1,34 @@
+general:
```

Comment:
Please remove all quotes where not required for better readability.
Quotes are only required on lines starting with a special character like `%`, `[`, `-` etc.
E.g., `input_dir: 'input_data'` can be `input_dir: input_data`.